AmplifyPlugins/Storage/Tests/AWSS3StoragePluginTests/AWSS3StoragePluginAsyncBehaviorTests.swift

```diff
@@ -11,24 +11,45 @@ import Amplify
 @testable import AmplifyTestCommon
 @testable import AWSPluginsTestCommon
 
-class AWSS3StoragePluginAsyncBehaviorTests: AWSS3StoragePluginTests {
+class AWSS3StoragePluginAsyncBehaviorTests: XCTestCase {
 
-    override func setUp() {
+    var storagePlugin: AWSS3StoragePlugin!
+    var storageService: MockAWSS3StorageService!
+    var authService: MockAWSAuthService!
+    var testKey: String!
+    var testURL: URL!
+    var testData: Data!
+    var queue: OperationQueue!
 
+    let defaultAccessLevel: StorageAccessLevel = .guest
+
+    override func setUpWithError() throws {
         storagePlugin = AWSS3StoragePlugin()
         storageService = MockAWSS3StorageService()
         authService = MockAWSAuthService()
-        let queue = OperationQueue()
-
+        testKey = UUID().uuidString
+        testURL = URL(fileURLWithPath: NSTemporaryDirectory().appendingPathComponent(UUID().uuidString))
+        testData = Data(UUID().uuidString.utf8)
+        queue = OperationQueue()
         storagePlugin.configure(storageService: storageService,
                                 authService: authService,
                                 defaultAccessLevel: defaultAccessLevel,
                                 queue: queue)
     }
 
+    override func tearDownWithError() throws {
+        queue.cancelAllOperations()
+
+        storagePlugin = nil
+        storageService = nil
+        authService = nil
+        testKey = nil
+        queue = nil
+    }
+
     func testPluginGetURLAsync() async throws {
```

Comment:
If it's OK with folks, I will submit a separate PR that describes these tests with Given/When/Then.